### PR TITLE
Input snippets

### PR DESCRIPTION
**Pull request by Chalarangelo**
Three simple components for passing data from form elements to the parent component:

* `Input` for `<input>`
* `TextArea` for `<textarea>`
* `Select` for `<select>`

### CODE REVIEW

File: snippets/Input.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+```jsx
+function Input ({ callback, type = 'text', disabled = false, readOnly = false, placeholder = '' }) {
+  return (
+    <input 
+      type={type} 
+      disabled={disabled} 
+      readOnly={readOnly} 
+      placeholder={placeholder}
+      onChange={(event) => callback(event.target.value)}
```
Comment (skatcat31) [inline snippets/Input.md]: You should use destructing:

```suggestion
      onChange={({ target: { value } }) => callback(value)} 
```

File: snippets/Select.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+```jsx
+function Select ({ values, callback, disabled = false, readonly = false, selected }) {
+  return (
+    <select 
+      disabled={disabled} 
+      readOnly={readonly} 
+      onChange={(event) => callback(event.target.value)} 
+    >
+      {values.map(v => <option selected={selected === v[0]}value={v[0]}>{v[1]}</option>)}
```
Comment (skatcat31) [inline snippets/Select.md]: You should use destrcuturing

```suggestion
      {values.map(([value, text]) => <option selected={selected === value}value={value}>{text}</option>)}
```

File: snippets/Select.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Render a `<select>` element with the appropriate attributes and use the `callback` function in the `onChange` event to pass the value of the textarea to the parent.
+Use the `values` array to pass an array of `[value, text]` elements and the `selected` attribute to define the initial `value` of the `<select>` element.
+
+```jsx
+function Select ({ values, callback, disabled = false, readonly = false, selected }) {
+  return (
+    <select 
+      disabled={disabled} 
+      readOnly={readonly} 
+      onChange={(event) => callback(event.target.value)}
```
Comment (skatcat31) [inline snippets/Select.md]: You should use desctructuring

```suggestion
      onChange={({ target : { value } }) => callback(value)} 
```

File: snippets/TextArea.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+```jsx
+function TextArea ({ callback, cols = 20, rows = 2, disabled = false, readOnly = false, placeholder='' }) {
+  return (
+    <textarea 
+      cols={cols}
+      rows={rows}
+      disabled={disabled} 
+      readOnly={readOnly} 
+      placeholder={placeholder}
+      onChange={(event) => callback(event.target.value)}
```
Comment (skatcat31) [inline snippets/TextArea.md]: You should use destructuring

```suggestion
      onChange={({ target : { value } }) => callback(value)} 
```

**Comment by skatcat31** [comment]
So I'm curious if we should update these snippet to just use callback(sometimes you actually want the event or the target), or if we want to keep them as is and should update them to use destructuring?

```js
const test = ({ test : { value } }) => console.log(value);

test({ test : { value : 'Hello World' } })
```

**Comment by skatcat31** [comment]
@Chalarangelo thoughts?

**Comment by Chalarangelo** [comment]
@skatcat31 I think we should go for the destructuring option. I'll update them as soon as I have some time.

**Comment by fejes713** [comment]
@Chalarangelo  no worries, I'll do so - I have some free time this morning 👍

**Comment by fejes713** [comment]
@Chalarangelo @skatcat31 resolved in #29

**Pull request merged**